.github/workflows/main.yaml

```diff
@@ -268,6 +268,30 @@ jobs:
           license-fields: ${{ env.LICENSE_FIELDS }}
           integration-enabled: 'false'
 
+      - name: Upgrade via Helm as subchart in production mode to a new version
+        run: |
+          oldpodname=$(kubectl get pods -l app.kubernetes.io/name=replicated -o jsonpath='{.items[0].metadata.name}')
+          
+          helm upgrade test-chart oci://registry.replicated.com/$APP_SLUG/$CHANNEL_SLUG/test-chart --set replicated.integration.enabled=false --set replicated.versionLabel=1.0.0 --wait --timeout 2m
+
+          COUNTER=1
```

Comment:
I found that `--wait` didn't always wait long enough so the test was flaky.  Here we will retry up to a timeout instead.